src/components/modules/blockManager.ts

```diff
@@ -647,18 +681,65 @@ export default class BlockManager extends Module {
   }
 
   /**
-   * Bind Events
+   * Bind Block events
    *
    * @param {Block} block - Block to which event should be bound
    */
-  private bindEvents(block: Block): void {
+  private bindBlockEvents(block: Block): void {
     const { BlockEvents, Listeners } = this.Editor;
 
-    Listeners.on(block.holder, 'keydown', (event) => BlockEvents.keydown(event as KeyboardEvent), true);
-    Listeners.on(block.holder, 'mousedown', (event: MouseEvent) => BlockEvents.mouseDown(event));
-    Listeners.on(block.holder, 'keyup', (event) => BlockEvents.keyup(event));
-    Listeners.on(block.holder, 'dragover', (event) => BlockEvents.dragOver(event as DragEvent));
-    Listeners.on(block.holder, 'dragleave', (event) => BlockEvents.dragLeave(event as DragEvent));
+    this.listenerIds.push(
+      Listeners.on(block.holder, 'keydown', (event: KeyboardEvent) => {
+        BlockEvents.keydown(event);
+      }, true)
+    );
+
+    this.listenerIds.push(
+      Listeners.on(block.holder, 'keyup', (event: KeyboardEvent) => {
+        BlockEvents.keyup(event);
+      })
+    );
+
+    this.listenerIds.push(
+      Listeners.on(block.holder, 'dragover', (event: DragEvent) => {
+        BlockEvents.dragOver(event);
+      })
+    );
+
+    this.listenerIds.push(
+      Listeners.on(block.holder, 'dragleave', (event: DragEvent) => {
+        BlockEvents.dragLeave(event);
+      })
+    );
+  }
+
+  /**
+   * Disable all handlers and bindings
+   * The sequence is following:
+   *  - Removes all listeners by id
+   *  - Removes all shortcuts
+   */
+  private disableModuleBindings(): void {
+    const { Listeners } = this.Editor;
+
+    for (const id of this.listenerIds) {
+      Listeners.offById(id);
+    }
```

Comment:
this.listener
```suggestion
    this.listenersIds.forEach(Listeners.offById);
```

Reply:
I think it is more readable with
```js
for (const id of this.listenerIds) {
      Listeners.offById(id);
}

this.listenerIds = [];
```

than

```js
this.listenersIds.forEach(Listeners.offById);
this.listenersIds = [];
```